app/web/src/newhotness/layout_components/AttributeInput.vue

```diff
@@ -634,16 +634,9 @@ const selectKeyString = "Tab";
 
 const anchorRef = ref<InstanceType<typeof HTMLElement>>();
 
-const path = computed(() => {
-  // fix the MV! for arrays path": "root\u000bdomain\u000btags\u000btag"
-  let path = props.path;
-  path = path.replaceAll("\u000b", "/");
-  return path;
-});
-
 type AttrData = { value: string };
 const wForm = useWatchedForm<AttrData>(
-  `component.av.prop.${props.attributeValueId}`,
+  `component.av.prop.${props.path}`,
```

Comment:
i think that ID is better here, b/c its unique per component, correct? E.g. if someone was editing the same path on two components it would overwrite one another... alternatively, stick the component ID in there

Reply:
oh, I see! yeah, I'll fix